src/node/rpc/network_identity_subsystem.h

```diff
@@ -26,5 +26,11 @@ namespace ccf
     {
       return network_identity;
     }
+
+    virtual std::optional<std::vector<uint8_t>> get_previous_identity_cert()
```

Comment:
Do we still need to call into the node state as the certificate is stored in the KV?

Reply:
Yes - this is where we get the value from that is then written to the KV.

Reply:
Why not just read it from the [`service.info`](https://microsoft.github.io/CCF/main/audit/builtin_maps.html#_CPPv4N3ccf11ServiceInfo4certE) table directly?